docusaurus/docs/what-is-included/available-scripts.md

```diff
@@ -92,6 +104,10 @@ This script will automatically open the documentation in your default browser.
 Keep in mind, you must have your documentation's dependencies installed before running this script!
 To to so, please run `npm install --prefix docusaurus` first.
 
+## `npm run prerelease`
+
+This script will run both `build:android:js` and `build:ios:js:prod` scripts, preparing for a release.
+
```

Comment:
This script is not supposed to be ran by the developer directly. As such, we can omit it.

Reply:
Ok, I will remove it 👍